src/bridgetree.rs

```diff
@@ -355,41 +374,51 @@ impl<H: Hashable + Clone, const DEPTH: u8> crate::Frontier<H> for Frontier<H, DE
     }
 }
 
+/// Each AuthFragment stores part of the authentication path for the leaf at a particular position.
+/// Successive fragments may be concatenated to produce the authentication path up to one less than
+/// the maximum altitude of the Merkle frontier corresponding to the leaf at the specified
+/// position. Then, the authentication path may be completed by hashing any 
```

Comment:
```suggestion
/// position. Then, the authentication path may be completed by hashing any 
```
I want to know what happens in the next episode!